osv/impact.py

```diff
@@ -105,17 +105,20 @@ def get_affected(self,
     return AffectedResult(tags, affected_commits, affected_ranges)
 
   def _get_affected_range(self,
-                          repo,
-                          regress_commits,
-                          fix_commits,
-                          limit_commits=None):
+                          repo: pygit2.Repository,
+                          regress_commits: list[str],
+                          fix_commits: list[str],
+                          limit_commits: list[str] = None):
     """Get affected range."""
     range_collector = RangeCollector()
     commits = set()
     seen_commits = set()
     tags = set()
     commits_to_tags = _get_commit_to_tag_mappings(repo)
     branch_to_limit = {}
+    repo_url = 'UNKNOWN_URL'
```

Comment:
nit: typically `None` is better to denote no value rather than a sentinel value.

Reply:
> nit: typically `None` is better to denote no value rather than a sentinel value.

That's a lot harder to definitively find in the logs to determine if it's even happening, though...

Reply:
The better practice would be to format this accordingly in the logs, rather than this potentially causing problems in the future because we have code that does `if repo_url: foo`. ,

Reply:
Fair enough, I think I've done as you advise in 1f2e404